src/executor.test.ts

```diff
@@ -284,20 +266,34 @@ describe('executor', () => {
         channelBalance.nonce,
       ]);
 
+      let incomingDelta: Int64;
+      let randomNumber: Field[];
       Circuit.runAndCheck(() => {
-        const incomingDelta = executor.flipCoin(
+        const randEncr = Encryption.encrypt([Field(100)], executorAddress);
+        const randSig = Signature.create(oraclePrivateKey, randEncr.cipherText);
+        [incomingDelta, randomNumber] = executor.flipCoin(
           player1PrivateKey.toPublicKey(),
           merkleMap.get(key),
           witness,
           channelBalance.deltaBalance,
           channelBalance.nonce,
-          channelBalanceSignature
+          channelBalanceSignature,
+          randSig,
+          randEncr.cipherText[0],
+          randEncr.cipherText[1],
+          randEncr.publicKey,
+          executorPrivateKey
         );
-        deltaBalance.add(incomingDelta);
+        console.log(`Incoming Delta Balance: ${incomingDelta.toString()}`);
+        deltaBalance = deltaBalance.add(incomingDelta);
         channelBalance.update(incomingDelta);
+        expect(randomNumber.toString()).toBe('100');
+        expect(incomingDelta.toString()).toBe('5');
```

Comment:
Test that the number we get back is the number we signed at the beginning.  That number is even, so the user won, so we expect `+5` to be the IOU.